addons/purchase_requisition/tests/common.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+from odoo.tests import common
+
+
+class TestPurchaseRequisitionCommon(common.TransactionCase):
+
+    def setUp(self):
+        super(TestPurchaseRequisitionCommon, self).setUp()
+
+        self.product_09_id = self.ref('product.product_product_9')
```

Comment:
Can you, like in sale tests, create 'setUp' method that are called in the `setUpClass` method to create product from scratch. We definitely need to stop using demo data in tests. So, please create method to generate the data you need in your test.

Reply:
Improve the tests like I describe in the review should be done a new separated commit ;)

Reply:
@jem-odoo  Changes done.